src/nvim/mouse.c

```diff
@@ -682,38 +682,39 @@ static int mouse_adjust_click(win_T *wp, int row, int col)
 
   vcol = offset;
 
-#define incr() nudge++; ptr_end += utfc_ptr2len(ptr_end)
-#define decr() nudge--; ptr_end -= utfc_ptr2len(ptr_end)
```

Comment:
think it's worth keeping these. They are just documentation. But they should be all uppercase.

Reply:
What do you mean by documentation? It's just a macro function? I'm reverting the change now, nothing against it, I'm just confused :).

Reply:
Names/symbols can/should serve the same purpose as documentation.

Reply:
OK, fixed :+1: